examples/Bremsstrahlung/README.rst

```diff
@@ -0,0 +1,44 @@
+Emission of Bremsstrahlung from Laser-foil interaction
+======================================================
+
+* author:      Heiko Burau <h.burau (at) hzdr.de>
+* maintainer:  Heiko Burau <h.burau (at) hzdr.de>
+
+This is a simulation of a flat solid density target hit head-on by a high-intensity laser pulse. 
+At the front surface free electrons are accelerated up to ultra relativistic energies and start travelling throught the bulk then. 
+Meanwhile, due to ion interaction, the hot electrons loose a small fraction of their kinectic energy in favor of emission of Bremsstrahlung-photons. 
+Passing over the back surface hot electrons are eventually reflected and re-enter the foil in opposite direction. 
+Because of the ultra-relativistic energy Bremsstrahlung (BS) is throughoutly emitted mainly along the direction of motion of the electron.
```

Comment:
~~throughoutly~~ (doesn't really exist *afaik*)
Leave it out? What did you want to say?